### DISCUSSION TITLE
Are you open to contributions providing more customization to the site generation?

### DISCUSSION THREAD

**Discussion by victorlevasseur** [avisi-cloud/structurizr-site-generatr#339] [General]
Hi,

My team is really interested in this site generator based on Structurizr. However, we might end up with a lot of software systems described in one website and it might produce a big website with a lot of links in the menu which can be quite difficult to navigate.

Are you open to contributions adding new customizations options (through new properties like https://github.com/avisi-cloud/structurizr-site-generatr#customizing-the-generated-website) ?

**Comment by qtzar** [comment]
There is a new feature coming in the next release that may help address this. 

#180 Nested Groups In Sidebar.

Basically I have the same issue with 100's of systems described in a single site. This feature, once released, will show the software systems nested using the groups{} feature of the structurizer DSL.

**Reply by victorlevasseur** [reply]
I'm really pleased to ear that. Thanks for the quick answer.

**Comment by qtzar** [comment]
@victorlevasseur The latest release now has the nested sidebar capability. I hope it does what you were hoping for.
